### DISCUSSION TITLE
Extend or switch theme runtime

### DISCUSSION THREAD

**Discussion by italo1983** [nuxt/nuxt#18753] [Questions]
I have a multi domain application and each user can choose a different theme for his frontend, the system should make an API call to fetch the settings from the backend and extend the theme in nuxt config. How could I change the extend path at runtime? I tried with before hook but I don't access the configuration inside defineNuxtConfig.

**Comment by TheAlexLichter** [comment]
You cannot change `extends` at runtime as it is required for build-time operations.
